features/step_definitions/images/image_create_steps.rb

```diff
@@ -30,12 +30,15 @@
 end
 
 When(/^I create new glance image based on jeos$/) do
+  imagedir = File.expand_path "~/cct-images/"
```

Comment:
This has a rather negative side effect of changing dir structure and taking space outside of the cloned repo dir. 
Additionally, cct can be run remotely from a user workstation, and this will create a new dir in his home dir. Wouldn't it be better to put it into `./tmp/` or `./tmp/cct-images` ?